index.html

```diff
@@ -1169,19 +1169,66 @@ <h2>Relative DID URLs</h2>
 
   </section>
 
-  <section class="informative">
+  <section>
     <h1>Data Model</h1>
-
+    <p>
+This specification defines an abstract data model for <a>DID documents</a> so it
+is independent of any specific representation. In this section we provide a
+high-level description of the data model, a set of requirements for
+representations, and a set of requirements for extensibility.
+    </p>
     <section>
       <h2>Definition</h2>
+      <p>
+A <a>DID document</a> consists of a collection of name-value pairs which will be
+referred to as properties in this document. The definitions of each of these
+properties are specified in section <a href="#core-properties"></a>. Specific
+representations are defined in section <a href="#core-representations"></a>.
+      </p>
     </section>
-
     <section>
-      <h2>Extensibility</h2>
+      <h2>Representations</h2>
+      <p>
+Following are the requirements for conformant representations.
+      </p>
+      <ol>
+        <li>
+A representation MUST define an unambiguous encoding and decoding of all
+properties defined in this specification such that the value of any property
+may be losslessly converted into another representation.
+        </li>
+        <li>
+The representation MUST have an unambiguous MIME type.
+        </li>
+        <li>
+The representation MUST define fragment processing rules for its MIME type that
```

Comment:
There's an issue https://github.com/w3c/did-core/pull/241 by @tplooker that asks if there is a guarantee that fragments will consistently identify the same parts of a DID document across different representations, or if each representation is free to use fragments in different ways (e.g. the spec currently mentions JSON Pointer, but that will probably not work in non-JSON-based representations).

Reply:
Made this change in commit [77a9787](https://github.com/w3c/did-core/pull/241/commits/77a97872959692d262c16bbd827f857677ba8f82) by specifying that every representation must follow the fragment processing rules defined in section 5.2.6.